### PR TITLE
Add Distributed Systems Conf

### PR DESCRIPTION
**Pull request by douglas-vaz**
Add available details for Distributed Systems Conference 2018.

**Details of the conference**:

- Website: https://dsconf.in/
- Date: 21st April 2018
- Venue: Pune
- Description: Topics on the best practices and future of distributed systems. Distributed Systems Pune Meetup is organising DSConf 2018.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,6 +11,7 @@ To add a conference to the list, please [contribute](#contributing)!
 | [JSFoo](https://jsfoo.in/2018/) | 13-14 Sep | Bengaluru | The eighth edition of India’s first JavaScript conference.
 | [PyCon India](https://in.pycon.org/2018/) | 22-26 Aug | Bengaluru International Exhibition Center (BIEC), Bengaluru | The premier conference in India on using and developing the Python programming language. |
 | [Great Indian Developer Summit](http://www.developermarch.com/developersummit/index.html) | 24-27 Apr | Bengaluru | 	With over 40000 attendees benefiting from ten game changing editions, GIDS is the gold standard for India's software professional ecosystem. |
+| [Distributed Systems Conference 2018](https://dsconf.in/) | 21 Apr | Pune | Best Practices and the Future of Distributed Systems. We want to help grow & highlight all the work in the areas of Distributed Systems. |
```
Comment (nikhita) [inline README.md]: For the description: `DSConf is a conference on best practices and future of distributed systems`.

**Comment by nikhita** [comment]
fyi @gouthamve

File: README.md (hunk 1 above)
Comment (nikhita) [reply README.md]: @douglas-vaz Can you rebase and fix the description? Thanks!

Happy to merge it once this is done.

**Comment by douglas-vaz** [comment]
Hi @nikhita , this branch has gone stale. Will close this and resubmit a new PR with updated details. Thanks!

**Pull request closed without merging**